server/settings/settings.go

```diff
@@ -135,6 +136,7 @@ func (s *Server) Get(ctx context.Context, q *settingspkg.SettingsQuery) (*settin
 		set.UiBannerPermanent = argoCDSettings.UiBannerPermanent
 		set.UiBannerPosition = argoCDSettings.UiBannerPosition
 		set.ControllerNamespace = s.mgr.GetNamespace()
+		set.UiDashboardLogo = argoCDSettings.UiDashboardLogo
```

Comment:
We should probably validate that this is either `http` or `https` schema before returning it.